From a114286c079c42067b001ac330501496e2e297a1 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Wed, 6 Sep 2017 11:53:02 +1000
Subject: [PATCH 46/47] test: fix compiler warning

signal-logging.c:182:12: warning: suggest parentheses around assignment used as truth value [-Wparentheses]

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
(cherry picked from commit ea82ececbf85a7ac3d0931687f44c57534fde17c)
---
 test/signal-logging.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/signal-logging.c b/test/signal-logging.c
index 3d2d04801..7f2972003 100644
--- a/test/signal-logging.c
+++ b/test/signal-logging.c
@@ -177,7 +177,7 @@ static void logging_format(void)
     strcpy(&buf[sizeof(buf) - 4], "end");
 
     LogInit(log_file_path, NULL);
-    assert(f = fopen(log_file_path, "r"));
+    assert((f = fopen(log_file_path, "r")));
 
 #define read_log_msg(msg) do {                                  \
         msg = fgets(read_buf, sizeof(read_buf), f);             \
-- 
2.14.2

